mungegithub/mungers/release-note-label.go

```diff
@@ -34,16 +34,17 @@ const (
 	releaseNote               = "release-note"
 	releaseNoteNone           = "release-note-none"
 	releaseNoteActionRequired = "release-note-action-required"
+	releaseNoteExperimental   = "release-note-experimental"
 
 	releaseNoteFormat = `Removing LGTM because the release note process has not been followed.
-One of the following labels is required %q, %q, or %q
-Please see: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes`
-	parentReleaseNoteFormat = `The 'parent' PR of cherry-picks must have either the %q or %q label or this PR must follow the release note process.`
+One of the following labels is required %q, %q, %q or %q.
+Please see: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes.`
+	parentReleaseNoteFormat = `The 'parent' PR of a cherry-pick PR must have one of the %q, %q or %q labels or this PR must then follow the standard/parent release note labeling requirement.`
 )
 
 var (
-	releaseNoteBody       = fmt.Sprintf(releaseNoteFormat, releaseNote, releaseNoteNone, releaseNoteActionRequired)
-	parentReleaseNoteBody = fmt.Sprintf(parentReleaseNoteFormat, releaseNote, releaseNoteActionRequired)
+	releaseNoteBody       = fmt.Sprintf(releaseNoteFormat, releaseNote, releaseNoteActionRequired, releaseNoteExperimental, releaseNoteNone)
+	parentReleaseNoteBody = fmt.Sprintf(parentReleaseNoteFormat, releaseNote, releaseNoteActionRequired, releaseNoteExperimental)
```

Comment:
Right. I would expect that a cherry-pick to a release branch where the parent was 'experimental' would be something we wouldn't automatically consider acceptable and we'd for the release branch owner to follow the release note process on the cherry-pick PR.  AKA I think you should revert https://github.com/kubernetes/contrib/pull/1111/files#diff-7294a858e5f70293980043d7fe211686R42
https://github.com/kubernetes/contrib/pull/1111/files#diff-7294a858e5f70293980043d7fe211686R94
https://github.com/kubernetes/contrib/pull/1111/files#diff-7294a858e5f70293980043d7fe211686R94